content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
 displayed_sidebar: platformSidebar
 ---
 
-<TopBlock>
+## Overview
 
-Platform Console enables you to integrate the latest Prisma Data Platform products into your application
+Platform Console enables you to integrate the latest Prisma Data Platform (PDP) products into your application
 
 - [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
 - [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
 
-[Go to Platform Console](https://console.prisma.io)
+Open the [Platform Console](https://console.prisma.io).
 
-</TopBlock>
+## Concepts
+
+The Platform Console workflows are based on four main concepts:
+
+- [**Account**](#account): In order to use PDP products, you need to have a PDP account. An _organization_ will typically create one account to manage all their workspaces, projects and environments.
```

Comment:
> An _organization_ will typically create one account to manage all their workspaces, projects and environments.

I don't understand this part, it kinda sounds like an org/company would create a shared account that everyone uses, which is not right. Am I missing something?

Reply:
Ah, never mind! Of course every member of the org will have their own account and there isn't a _shared_ account for everyone across the company. I see how my phrasing is confusing here (I've even confused myself 😅 ), will try to make it clearer!

Reply:
How about this:

- [**Account**](#account): In order to use PDP products, you need to have a PDP account. A _user_ will typically create one account to manage all their workspaces, projects and environments. The _user_ can also be invited to join other workspaces to collaborate on the projects in that workspace.

Reply:
Yep, you found the confusing parts 😄 I'd just call this a user, vs. an account. I think it makes things more clear. You also have a user ID, and you have user settings, etc.